Fix #1727 -- Retire django-developers and django-users mailing list

ontowhee opened the pull request:
Fix for #1727 

I've removed the references to the mailing lists. I hope it looks good!

### Footer

Before
![footer-mailing-list-before](https://github.com/user-attachments/assets/d5e89286-ecd6-4445-8185-350d9fcf3a06)

After
![footer-mailing-list-after](https://github.com/user-attachments/assets/500f4220-f726-4b39-a5da-a49ddf731ba0)

### Community

Before
![community-mailing-list-before](https://github.com/user-attachments/assets/37fb47c6-4fd1-4fe5-afe3-fa0f32437b7c)

After
![community-mailing-list-after](https://github.com/user-attachments/assets/82cb2d3b-cc6c-4f1b-a17b-e67a3f9c2260)

### Download page

Before
![download-mailing-list-before](https://github.com/user-attachments/assets/ec5c9763-97b1-4385-86dc-b893359b5369)

After
![download-mailing-list-after](https://github.com/user-attachments/assets/2ab40ca7-bdda-4701-91cf-7ae0353d4d30)

### Homepage

Before
![homepage-mailing-list-before](https://github.com/user-attachments/assets/95347813-4e48-44be-accd-c27c8ab8313e)

After
![homepage-mailing-list-after](https://github.com/user-attachments/assets/4e48635f-7eb3-407e-9f11-a0917385dc5b)

### Contact Foundation

Before
<img width="750" alt="foundation-before" src="https://github.com/user-attachments/assets/10917597-94bc-41b2-8672-5e773062a0f2">

After
<img width="772" alt="foundation-after" src="https://github.com/user-attachments/assets/5c623d14-6e4e-4bbe-9fa9-0f31c20636b9">

### Docs Template Sidebar

Before
<img width="429" alt="sidebar-before" src="https://github.com/user-attachments/assets/263f39fe-2b9c-4ef7-9632-c75213e1d7cd">

After
<img width="430" alt="sidebar-after" src="https://github.com/user-attachments/assets/b2318198-2311-4bc9-b767-d1e874f13306">

bmispelon commented on djangoproject/templates/aggregator/index.html:
```diff
@@ -84,8 +84,4 @@ <h2 class="bullet-icon"><i class="icon icon-rss blue"></i> {{ feedtype.name }}</
   </ul>
-
-  <h2>{% translate "Mailing Lists" %}</h2>
-  {% include "includes/mailing_lists.html" %}
```
This included template is not used anymore now and can be deleted

ontowhee replied:
Thanks for pointing this out! It is now deleted!

bmispelon commented on djangoproject/templates/releases/download.html:
```diff
@@ -59,7 +59,7 @@ <h2>Option {% cycle options %}: Get the latest development version</h2>
   <h2>After you get it</h2>
   <p>See the <a href="{% url 'document-detail' lang='en' version='stable' url='intro/install' host 'docs' %}">installation guide</a> for further instructions. Make sure you read the documentation that corresponds to the version of Django you’ve just installed.</p>
-  <p>And be sure to sign up for the <a href="https://groups.google.com/group/django-users">django-users mailing list</a>, where other Django users and the Django developers themselves all hang out to help each other.</p>
+  <p>And be sure to sign up to the <a href="https://forum.djangoproject.com">Django Forum</a>, where other Django users and the Django developers themselves all hang out to help each other.</p>
```
Nice rewrite 👍🏻

bmispelon reviewed (changes requested):
This is a great start, thanks for working on this 🙇🏻 

Using `git grep mailing.list` I found more places that should be fixed:
- [x] `djangoproject/templates/contact/foundation.html`
- [x] `docs/templates/docs/doc.html`

There's also things inside `djangoproject/templates/conduct/` that should be updated, but I don't know if that requires approval from the CoC work group. I'll ask around, we can do it in a separate PR later.

ontowhee commented:
I've updated `djangoproject/templates/contact/foundation.html` and `docs/templates/docs/doc.html`. Thanks for mentioning these! I've added their screenshots to the PR description.

bmispelon reviewed (approved):
This is looking great, nice work 👍🏻 

Thanks for taking the time to take screenshots too, it made reviewing even easier 🙇🏻

thibaudcolas commented:
Thank you! 😌 Now we have to figure out what to do with all that space on the homepage!
